website/www/site/content/en/documentation/sdks/python-machine-learning.md

```diff
@@ -47,9 +47,16 @@ For more infomation about machine learning with Apache Beam, see:
 * [About Beam ML](/documentation/ml/about-ml)
 * [RunInference notebooks](https://github.com/apache/beam/tree/master/examples/notebooks/beam-ml)
 
+## Support and limitations
+
+- The RunInference API is supported in Apache Beam 2.40.0 and later versions.
+- PyTorch and Scikit-learn frameworks are supported. Tensorflow models are supported through tfx-bsl.
```

Comment:
This is out dated. we no longer need tfx-bsl. Lets just change this to Tensorflow, Pytorch and other frame works are supported, for a full list look ...

Reply:
Updated, but the update should be verified.